assets/js/truelayer-for-woocommerce.js

```diff
@@ -0,0 +1,55 @@
+/* global truelayerParams */
+jQuery(function ($) {
+    var truelayerForWooCommerce = {
+        Truelayer: null,
+
+        Payment: null,
+
+        init: function () {
+            window.addEventListener("hashchange", truelayerForWooCommerce.handleHashChange);
```

Comment:
We already have webpack in place, can we use `import` from the EPP package rather than relying on the global window variable?

Reply:
Yes, we can do that. We did not do that to start with since that removes the ability to use the non-minified script when WordPress config is set to [script_debug](https://wordpress.org/documentation/article/debugging-in-wordpress/#script_debug). But webpack will create a map anyway, so that should be able to be used instead.

The last commits have made this change to this script and the admin script as well.